pages/osx/spotify.md

```diff
@@ -0,0 +1,24 @@
+# shpotify
+
+> A command-line interface to Spotify.
+> More information: <https://github.com/hnarayanan/shpotify>.
+
+- Find a song by name and play it:
+
+`spotify play {{song name}}`
```

Comment:
I would say we use snake case here but that might lead to confusion so I think it is good the way it is.